### ISSUE TITLE
Question QA Performance

### ISSUE THREAD

**Issue by vthost** [MaHuanAAA/logtoku#2]
Hi, 
This is a very interesting paper!
I tried the code to estimate LLM accuracy on a few common question answering datasets (e.g., TriviaQA, CoQA) but get very mixed results. Some scores are good but some look random. Did you maybe try that as well or do you have an idea how to set the parameters to get more reasonable results?
Thank you!

**Comment by MaHuanAAA** [comment]
Very excited upon receiving the email notification
We have already assigned our master's students to evaluate on these datasets. Failure cases often imply new discoveries. Based on feedback and discussions from the community, we have already updated the paper four times. If things are truly as you described, it means we now have new patterns and corresponding explanations worth exploring.

**Comment by MaHuanAAA** [comment]
Before that, could you share the average length of the answers on your setting? If the answers are very short, it might indicate that the responses on these datasets are definitional, with almost no cases falling into the fourth quadrant as discussed in the paper. In such cases, the improvement brought by uncertainty decoupling may not be noticeable.

**Comment by vthost** [comment]
Thank you! Yes, I understand that it's different with definitional data, but you would expect some benefits if there is either aleatoric or epistemic uncertainty, no? But yes, I agree, we need to research more :)

Oh, would you mind to recommend K-parameters you would try? I fixed K1 to 2 since this seems to be used in the code and was checking 5, 10 and 25 for K2 according to the Table in the appendix / code.

**Comment by MaHuanAAA** [comment]
> Thank you! Yes, I understand that it's different with definitional data, but you would expect some benefits if there is either aleatoric or epistemic uncertainty, no? But yes, I agree, we need to research more :)
> 
> Oh, would you mind to recommend K-parameters you would try? I fixed K1 to 2 since this seems to be used in the code and was checking 5, 10 and 25 for K2 according to the Table in the appendix / code.

<html><body>
<ul>
Since the main improvement of LogTokU over previous methods lies in 
its ability to identify more than one answer, setting it to 2 can avoid the instability caused by incorporating more token logits. Since setting it to 2 is already optimal, this parameter is not sensitive to changes. The table below compares the performance when the parameter is set to 5.

Metric | llama2-7b | llama2-13b | llama2-70b | llama3.2-3b | llama3.1-8b | llama3.1-70b
-- | -- | -- | -- | -- | -- | --
Probability | 57.00 | 69.11 | 57.92 | 69.07 | 73.08 | 36.49
Entropy | 60.90 | 74.38 | 60.79 | 70.24 | 74.90 | 39.14
LogTokU_2 | 71.06 | 80.02 | 62.05 | 69.49 | 81.55 | 44.89
LogTokU_5 | 67.99 | 77.29 | 61.53 | 65.73 | 81.49 | 41.01


</body>
</html>

**Comment by MaHuanAAA** [comment]
> Hi, This is a very interesting paper! I tried the code to estimate LLM accuracy on a few common question answering datasets (e.g., TriviaQA, CoQA) but get very mixed results. Some scores are good but some look random. Did you maybe try that as well or do you have an idea how to set the parameters to get more reasonable results? Thank you!

Yesterday I arranged the experiments, and now I have obtained the results. The performance on TriviaQA is indeed mixed, while there is a clear improvement on BIOASQ. I suspect that the answers might be too short, making the differences less noticeable—similar to how robust optimization doesn't show significant improvement over the baseline on clean test sets. Of course, this is only a hypothesis for now, and it's too early to draw any conclusions. As for effectiveness, based on application feedback, the improvement seems quite evident. We will continue to monitor this and there may be other possible explanations.
